packages/interactivity/src/store.js

```diff
@@ -38,6 +38,47 @@ const getSerializedState = () => {
 const rawState = getSerializedState();
 export const rawStore = { state: deepSignal( rawState ) };
 
+/**
+ * Extends the Interactivity API global store with the passed properties.
+ *
+ * These props tipically consist of `state`, `selectors` and `actions` that can
+ * be used by any directive, making the HTML reactive and interactive.
```

Comment:
Maybe we can separate `state` from the others and mention that it is reactive.

```suggestion
 * These props typically consist of `state`, which is reactive, and other
 * properties like `selectors`, `actions`, `effects`, etc. which can store
 * callbacks and derived state. These props can then be referenced by any
 * directive to make the HTML interactive.
```